lib/importeur/data_sources/appnexus.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Importeur
+  module DataSources
+    class Appnexus
+      def initialize(appnexus_service)
+        @appnexus_service = appnexus_service
```

Comment:
How about adding a second parameter, `params` that we then pass to `get_all` call bellow? This could be useful to limit number of rows returned in tests (reducing VCR file size), for example: 

````ruby
Importeur::DataSources::Appnexus.new(service, num_elements: 1)
````

Reply:
Wouldn't `get_all` itself be a better place for that?

Reply:
`get_all` already support passing params https://github.com/simplifi/appnexusapi/blob/f5680b8f4df443d9fd1f0e6c1478418fd79eb774/lib/appnexusapi/service.rb#L52

So what I thought is that Appnexus DataSource could support passing `params` to `get_all`.

Reply:
I got that, I just don't understand why. For testing?

Reply:
Yea, I thought of that when I see this: https://github.com/ad2games/importeur/pull/2/files#diff-af2edb5a6f6126be89f58a76ff77b1e4R43

But I imagine it could be useful if the user of Appnexus Datasource want to filter or something.

Another thing I just noted looking at get_all source code: it implements pagination and will only return first 100 records by default. So if the Datasource should be able to get more than 100 records, we need to call get_all multiple times (or pass a bigger num_elements, but this don't really solve the problem)

Reply:
It would add more code and hence more things that can break, only to make it more testable. I'm not a fan.

Are you talking about [appnexus_api](https://github.com/simplifi/appnexusapi/blob/master/lib/appnexusapi/service.rb#L52-L62)? It calls `get` while incrementing `start_element` until the result is empty.

Reply:
Oh, sorry, yes, it already does that, nevermind.

Regarding the params, I think it could be useful to pass filters, beside testing with num_elements: 1. But I agree that we can postpone this until/if it is really needed and not do that just for testing.